frontend/public/components/dashboard/dashboard-card/dashboard-card-title-see-all.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import { Button, OverlayTrigger, Popover } from 'patternfly-react';
+
+const SEE_ALL = 'See all';
```

Comment:
@spadgett , isn't there a better place where to store strings all-together?

Reply:
> @spadgett , isn't there a better place where to store strings all-together?

We don't currently externalize any strings, so there's no good place at the moment.